tf_files/kube.tf

```diff
@@ -0,0 +1,235 @@
+
```

Comment:
We should have a style for organizing these so that like resource blocks end up near each other